3.7/icegridnode/Dockerfile

```diff
@@ -1,20 +1,40 @@
-FROM ubuntu:16.04
+FROM ubuntu:22.04
 
 LABEL maintainer="docker-maintainers@zeroc.com"
 
-ARG ICEGRID_VERSION=3.7.8
-
+ARG ICEGRID_VERSION=3.7.9
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv B6391CB2CFBA643D \
-    && echo "deb http://download.zeroc.com/Ice/3.7/ubuntu16.04 stable main" > /etc/apt/sources.list.d/ice.list \
-    && apt-get update \
+RUN groupadd -r ice && useradd -r -m -d /var/lib/ice -g ice ice
+
+RUN apt-get update && apt-get install --no-install-recommends --no-install-suggests -y \
+    ca-certificates \
+    curl \
+    gpg \
+    gosu \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN mkdir -p /etc/apt/keyrings \
+    && curl -fsSL https://zeroc.com/download/GPG-KEY-zeroc-release-B6391CB2CFBA643D |  gpg --dearmor -o /etc/apt/keyrings/zeroc.gpg \
+    && echo \
+    "deb [signed-by=/etc/apt/keyrings/zeroc.gpg] https://zeroc.com/download/ice/3.7/ubuntu22.04 stable main" | tee /etc/apt/sources.list.d/zeroc-ice-3.7.list
+
+RUN apt-get update \
     && apt-get install --no-install-recommends --no-install-suggests -y \
     zeroc-icegrid=${ICEGRID_VERSION}-* \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+RUN sed -i '/^Ice.UseSystemdJournal=1/d' /etc/icegridnode.conf \
+    && sed -i 's/^IceGrid\.Node\.Data=.*/IceGrid\.Node\.Data=\/var\/lib\/ice\/icegrid/' /etc/icegridnode.conf \
+    && sed -i 's/^IceGrid\.Node\.Output=.*/#IceGrid\.Node\.Output=\/var\/lib\/ice\/icegrid/' /etc/icegridnode.conf \
+    && sed -i 's/^Ice.Default.Locator=.*/#Ice.Default.Locator=/' /etc/icegridnode.conf \
```

Comment:
Why is this required?

Reply:
The default that we set in the rpm is relative to the default name `node1`, but you would never have two instances running int he container. 

```
IceGrid.Node.Output=/var/lib/ice/icegrid/node1
```